pkgs/applications/networking/qv2ray/default.nix

```
@@ -0,0 +1,71 @@
+{ stdenv
+, fetchFromGitHub
+, qmake
+, qttools
+, wrapQtAppsHook
+, cmake
+, clang
+, grpc
+, protobuf
+, openssl
+, pkgconfig
+, c-ares
+, abseil-cpp
+, libGL
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qv2ray";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Qv2ray";
+    repo = "Qv2ray";
+    rev = "v${version}";
+    sha256 = "12x72rsd5iaj06qmma8x6g7syqaiixnfb37f3ppx8x00c77s78va";
+    fetchSubmodules = true;
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DQV2RAY_DISABLE_AUTO_UPDATE=on"
+    "-DQV2RAY_TRANSLATION_PATH=${placeholder "out"}/share/qv2ray/lang"
+  ];
+
+  preConfigure = ''
+    export _QV2RAY_BUILD_INFO_="Qv2ray Nixpkgs"
+    export _QV2RAY_BUILD_EXTRA_INFO_="(Nixpkgs build) nixpkgs"
+  '';
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
+    libGL
+    zlib
+    grpc
+    protobuf
+    openssl
+    abseil-cpp
+    c-ares
+  ];
+
+  nativeBuildInputs = [ 
+    cmake
+    clang
+    pkgconfig
+    qmake
+    qttools
+    wrapQtAppsHook
+  ];
+
+  meta = with stdenv.lib; {
+    description = "An GUI frontend to v2ray";
+    homepage = https://qv2ray.github.io/en/;
```

```suggestion
    homepage = "https://qv2ray.github.io/en/";
```
due to https://github.com/NixOS/rfcs/pull/45/files

---

this is still relevant